src/plugins/data/public/ui/filter_bar/_global_filter_group.scss

```diff
@@ -1,6 +1,7 @@
 // SASSTODO: Probably not the right file for this selector, but temporary until the files get re-organized
 .globalQueryBar {
   padding: 0 $euiSizeS $euiSizeS $euiSizeS;
+  height: 160px;
```

Comment:
This line should not be added. This makes dashboard and visualize top nav bar being displayed incorrectly. 
![Screenshot 2024-07-23 at 12 55 25 PM](https://github.com/user-attachments/assets/1dc0ee17-cc1c-4af5-8e32-b7dd9daa89b6)
![Screenshot 2024-07-23 at 12 55 15 PM](https://github.com/user-attachments/assets/b7d16e8c-ca33-4fe9-9e55-8fb1cebc450a)